upup/pkg/fi/cloudup/tagbuilder.go

```diff
@@ -78,6 +78,11 @@ func buildCloudupTags(cluster *api.Cluster) (sets.String, error) {
 	case api.CloudProviderBareMetal:
 		// No tags
 
+	case api.CloudProviderOpenstack:
+		{
+			tags.Insert("_openstack")
```

Comment:
So hopefully we don't need any tags for openstack - this is the very legacy way of selecting tasks stored in files, and we're _almost_ completely rid of them now.  But this tag isn't doing any harm :-)

Reply:
got it. I will update it.